autogen/templates/cpp_generic_report_status.liquid

```diff
@@ -1,7 +1,9 @@
 {% for prop in currentClass.systemProperties %}{%
   assign cppName = prop.name | downcase | underscore_spaces %}{%
   if prop.readAccess == true %}
-         << "    {{ prop.name }}: " << dev.{{ cppName }}() << endl{%
+    cout << "    {{ prop.name }}: ";
+    try { cout << dev.{{ cppName }}() << endl; }
+    catch(...) { cout << "[not supported by the device]" << endl; }{%
```

Comment:
"Not supported" is not the only possible error here. For example, you could get `-EINVAL` if an argument is invalid or a number of other errors. I would think it would be better to use `strerror()` to get the actual error message.